examples/quickstarts/cakephp-mysql-persistent.json

```diff
@@ -16,7 +16,8 @@
   },
   "message": "The following service(s) have been created in your project: ${NAME}, ${DATABASE_SERVICE_NAME}.\n\nFor more information about using this template, including OpenShift considerations, see https://github.com/openshift/cake-ex/blob/master/README.md.",
   "labels": {
-    "template": "cakephp-mysql-persistent"
+    "template": "cakephp-mysql-persistent",
+    "app": "cakephp-mysql-persistent"
```

Comment:
The app label should be parameterized so that when templates are used in conjunction to form a single application we can make the `app` labels match.